packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -25,12 +25,16 @@ export async function tailLogGroup(
     if (!wizardResponse) {
         throw new CancellationError('user')
     }
-
+    const awsCredentials = await globals.awsContext.getCredentials()
```

Comment:
I'm not familiar with how this works under the hood. Will it fall back to default credentials if nobody is logged into the toolkit? Or does it require someone to be logged in? If so, maybe we should re-direct them to login with credentials if they do not exist?

Reply:
If you have no selected credential, the explorer side bar will prompt you to add/select one. 
<img width="1064" alt="Screenshot 2024-11-13 at 10 39 36 AM" src="https://github.com/user-attachments/assets/26c1b3d6-3572-40a0-9e9a-dd0573c6a7fe">

If the selected credential is expired, it will say that, and give you an option to re-authenticate.
<img width="535" alt="Screenshot 2024-11-13 at 10 40 49 AM" src="https://github.com/user-attachments/assets/28e3eb29-3061-40ca-8387-d9dab4ae0cb0">

If your credential is expired/unselected and you try to select TailLogGroup from the command palette, this first picker to select a LogGroup will show `[Error loading Items] Toolkit is not logged in.`
<img width="641" alt="Screenshot 2024-11-13 at 10 39 15 AM" src="https://github.com/user-attachments/assets/65b77d3a-9383-4072-a21c-3262d02952e2">

Reply:
why not do this implicitly in your client, instead of plumbing it through?